das/dasrpc/dasRpcClient.go

```diff
@@ -35,15 +35,15 @@ func NewDASRPCClient(target string) (*DASRPCClient, error) {
 	}, nil
 }
 
-func (c *DASRPCClient) GetByHash(ctx context.Context, hash []byte) ([]byte, error) {
+func (c *DASRPCClient) GetByHash(ctx context.Context, hash common.Hash) ([]byte, error) {
 	if len(hash) != 32 {
 		return nil, fmt.Errorf("Hash must be 32 bytes long, was %d", len(hash))
 	}
 	var ret hexutil.Bytes
-	if err := c.clnt.CallContext(ctx, &ret, "das_getByHash", hexutil.Bytes(hash)); err != nil {
+	if err := c.clnt.CallContext(ctx, &ret, "das_getByHash", hexutil.Bytes(hash[:])); err != nil {
```

Comment:
You might be able to replace `hexutil.Bytes(hash[:])` with just `hash`. I think those will both get turned into a 0x prefixed hash in the RPC request

Reply:
Good point, switched